src/style.css

```diff
@@ -0,0 +1,20 @@
+body {
+  margin: 0;
+}
+
+div {
+  width: 100%;
```

Comment:
Block elements has 100% of the width of the parent container. Set width of 100% is redundant